src/assets/_project/_blocks/franchise/DFV/qg-dfv.scss

```diff
@@ -1,12 +1,24 @@
-.dfv-content a.qg-links-list__link, .dfv-back a {
+.dfv-content a.qg-links-list__link, .dfv-back a, a.qg-aside-button {
     @include qg-link-styles__theme-white;
 }
 .dfv-content a:visited:not(.btn, .button, .qg-btn), a.dfv-resource-nav__link {
     color: currentColor;
 }
-.dfv-back a {
+.dfv-back a, a.qg-aside-button {
     text-decoration: none !important;
+    font-size: 1rem;
+    font-weight: 700 !important;
+}
+.dfv-back span, .dfv-back a, a.qg-aside-button {
+    &:hover {
+        text-decoration: underline !important;
+        text-decoration-thickness: 2px !important;
+    }
 }
 #qg-primary-content .dfv-page-introduction a, #qg-primary-content .dfv-content a:not(.qg-links-list__link) {
     color: $qg-blue-dark !important;
 }
+
+#qg-content #qg-primary-content .dfv-topics__link {
+    margin: 0 !important;
```

Comment:
Why does this need to be `!important`?

Reply:
This is to override the default 2px margin. I can replace it with `@include qg-global-link-styles($margin: 0);`
I'll update it soon.